hydra-tui/test/Hydra/TUISpec.hs

```diff
@@ -40,61 +40,65 @@ import Hydra.ContestationPeriod (toNominalDiffTime)
-          shouldRender "Idle"
-          sendInputEvent $ EvKey (KChar 'i') []
-          threadDelay 1
-          shouldRender "Initializing"
-          sendInputEvent $ EvKey (KChar 'a') []
-          threadDelay 1
-          shouldRender "Idle"
-          sendInputEvent $ EvKey (KChar 'q') []
+        it "supports the init & abort Head life cycle" $
+          \TUITest{sendInputEvent, shouldRender} -> do
+            threadDelay 1
+            shouldRender "connected"
+            shouldRender "Idle"
+            sendInputEvent $ EvKey (KChar 'i') []
+            threadDelay 1
+            shouldRender "Initializing"
+            sendInputEvent $ EvKey (KChar 'a') []
+            threadDelay 1
+            shouldRender "Idle"
+            sendInputEvent $ EvKey (KChar 'q') []
 
-      it "supports the full Head life cycle" $
-        \TUITest{sendInputEvent, shouldRender} -> do
-          threadDelay 1
-          shouldRender "connected"
-          shouldRender "Idle"
-          sendInputEvent $ EvKey (KChar 'i') []
-          threadDelay 1
-          shouldRender "Initializing"
-          sendInputEvent $ EvKey (KChar 'c') []
-          threadDelay 1
-          shouldRender "42000000 lovelace"
-          sendInputEvent $ EvKey (KChar ' ') []
-          sendInputEvent $ EvKey KEnter []
-          threadDelay 1
-          shouldRender "Open"
-          sendInputEvent $ EvKey (KChar 'c') []
-          threadDelay 1
-          shouldRender "Closed"
-          shouldRender "Remaining time to contest"
-          threadDelay (realToFrac $ toNominalDiffTime tuiContestationPeriod + gracePeriod + someTime)
-          sendInputEvent $ EvKey (KChar 'f') []
-          threadDelay 1
-          shouldRender "Final"
-          shouldRender "42000000 lovelace"
-          sendInputEvent $ EvKey (KChar 'q') []
+        it "supports the full Head life cycle" $
+          \TUITest{sendInputEvent, shouldRender} -> do
+            threadDelay 1
+            shouldRender "connected"
+            shouldRender "Idle"
+            sendInputEvent $ EvKey (KChar 'i') []
+            threadDelay 1
+            shouldRender "Initializing"
+            sendInputEvent $ EvKey (KChar 'c') []
+            threadDelay 1
+            shouldRender "42000000 lovelace"
+            sendInputEvent $ EvKey (KChar ' ') []
+            sendInputEvent $ EvKey KEnter []
+            threadDelay 1
+            shouldRender "Open"
+            sendInputEvent $ EvKey (KChar 'c') []
+            threadDelay 1
+            shouldRender "Closed"
+            shouldRender "Remaining time to contest"
+            threadDelay (realToFrac $ toNominalDiffTime tuiContestationPeriod + gracePeriod + someTime)
+            sendInputEvent $ EvKey (KChar 'f') []
+            threadDelay 1
+            shouldRender "Final"
+            shouldRender "42000000 lovelace"
+            sendInputEvent $ EvKey (KChar 'q') []
+    it "should format time with whole values for every unit, not total values" $ do
```

Comment:
You could surround this function with a `describe "renderText"` to indicate its about that function here. Then the context would read `/TUI/renderText/should format time with whole values for every unit, not total values`

Should: surround unit test with function name in a `describe` or `context`